tests/integration/basic_data/program/resource.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
```

Comment:
It doesn't look like this is called from `program/index.ts`. Is this still necessary?